channeldb/invoices.go

```diff
@@ -2384,14 +2427,20 @@ func (d *DB) cancelInvoice(invoices kvdb.RwBucket, invoiceNum []byte,
 		}
 	}
 
-	err = d.serializeAndStoreInvoice(invoices, invoiceNum, invoice)
+	err = d.cancelInvoiceStoreUpdate(invoices, invoiceNum, invoice)
 	if err != nil {
 		return nil, err
 	}
 
 	return invoice, nil
 }
 
+func (d *DB) cancelInvoiceStoreUpdate(invoices kvdb.RwBucket, invoiceNum []byte,
+	invoice *invpkg.Invoice) error {
+
```

Comment:
Not much distance between this and the old call site?

Reply:
This is really just a boring preliminary step to separate in-memory vs storage stuff.